app/models/translation.rb

```diff
@@ -28,4 +28,12 @@ def blank?
   def to_s
     text
   end
+
+  def empty?
```

Comment:
What are these two methods for?

Reply:
they were required for simple format to work in the translatable modal pretty much like how the other 2 methods are used

Reply:
Ok. Thanks.